scram-tools.file/tools/gcc/env.sh

```diff
@@ -34,7 +34,7 @@ source ${SCRAM_TOOLS_BIN_DIR}/os_libdir.sh
 # optimizations as they become available in gcc.
 
 GCC_CXXFLAGS=""
-GCC_CXXFLAGS="$GCC_CXXFLAGS -std=c++20 -ftree-vectorize"
+GCC_CXXFLAGS="$GCC_CXXFLAGS -std=${CMS_CXX_STANDARD} -ftree-vectorize"
```

Comment:
I guess we are missing `c++` here i.e. it should be `-std=c++${CMS_CXX_STANDARD}` ... right?